pkg/reconciler/v1alpha1/builder/builder.go

```diff
@@ -122,16 +122,16 @@ func (c *Reconciler) reconcileBuilderStatus(builder *v1alpha1.Builder) reconcile
 				ObservedGeneration: builder.Generation,
 				Conditions: duckv1alpha1.Conditions{
 					{
-						Type:   duckv1alpha1.ConditionReady,
-						Status: corev1.ConditionFalse,
+						Type:    duckv1alpha1.ConditionReady,
+						Status:  corev1.ConditionFalse,
+						Message: err.Error(),
 					},
 				},
 			},
 		}
 
 		return reconciledBuilderResult{
```

Comment:
If we never return an error anymore... Let's remove the reconciledBuilderResult.